packages/x-date-pickers/src/DateField/tests/editing.DateField.test.tsx

```diff
@@ -275,7 +275,28 @@ describe('<DateField /> - Editing', () => {
           });
         });
 
-        it('should only call `onChange` when clearing the last section', () => {
+        it('should not call `onChange` when clearing all sections and both dates are already empty', () => {
+          const handleChange = spy();
+
+          render(
+            <DateField
+              format={adapter.formats.monthAndYear}
+              defaultValue={null}
+              onChange={handleChange}
+            />,
+          );
+
+          const input = getTextbox();
+          clickOnInput(input, 1);
+
+          // Select all sections
+          userEvent.keyPress(input, { key: 'a', ctrlKey: true });
+
+          userEvent.keyPress(input, { key: keyToClearValue });
+          expect(handleChange.callCount).to.equal(0);
+        });
+
+        it('should call `onChange` when clearing each section', () => {
```

Comment:
After the behavior change this test becomes somewhat misleading or incomplete. 🤔 
We won't call `onChange` after **each** section is cleared if there are more than 2 sections.
Maybe it's worth to also check what kind of value or validation error is returned after each action on top of checking call count? 🤔

Reply:
Done :+1: